pdns/stubresolver.cc

```diff
@@ -80,6 +104,16 @@ void stubParseResolveConf()
 // s_resolversForStub contains the ComboAddresses that are used to resolve the
 int stubDoResolve(const DNSName& qname, uint16_t qtype, vector<DNSZoneRecord>& ret)
 {
+  // only check if resolvers come from he local resolv.conf in the first place
+  if (s_localResolvConfMtime != 0) {
+    struct stat st;
+
+    if (stat(LOCAL_RESOLV_CONF_PATH, &st) != -1) {
+      if (st.st_mtime > s_localResolvConfMtime) {
```

Comment:
What about using `!=` instead of `>`? It's a bit weird, but someone could revert to an older backup copy of the file or something.